components/core/ContentFragmentList.js

```diff
@@ -0,0 +1,50 @@
+const AbstractComponent = require('../AbstractComponent');
+
+/**
+ * The Content Fragment List Core Component. In addition to the Component methods, this component provides several 
+ * convience methods to set known properties. 
+ * 
+ * Based on the documentation found at:
+ * https://github.com/adobe/aem-core-wcm-components/tree/master/content/src/content/jcr_root/apps/core/wcm/components/contentfragmentlist/v1/contentfragmentlist
+ */
+class ContentFragmentList extends AbstractComponent {
+
+    constructor() {
+        super();
+        this.props["sling:resourceType"] = "core/wcm/components/contentfragmentlist/v1/contentfragmentlist";
+    }
+
+    setModalPath(modalPath) {
+        this.props["modalPath"] = modalPath;
+    }
+
+    setParentPath(parentPath) {
+        this.props["parentPath"] = parentPath;
+    }
+
+    setTagNames(tagNames) {
```

Comment:
This this one is potentially interesting... we'll sync tomorrow. This is a multi-value prop. This can be potentially addressed by doing the following: 
```javascript
setTagNames(tagNames) {
  this.setMultiValueProp("tagNames", tagNames); 
}
```
To validate this value, you'll want to use this component in a page and check how the values are stored.